test/ruby/test_range.rb

```diff
@@ -127,6 +127,10 @@ def test_max
     assert_raise(RangeError) { (1...).max(3) }
 
     assert_raise(RangeError) { (..0).min {|a, b| a <=> b } }
+
+    assert_equal(2, (..2).max)
+    assert_raise(TypeError, (...2).max)
+    assert_raise(TypeError, (...2.0).max)
```

Comment:
The code to raise must be inside the block.

```suggestion
    assert_raise(TypeError) {(...2).max}
    assert_raise(TypeError) {(...2.0).max}
```

Reply:
Yes, sorry, did this at night without checking/running as updating the test script was an afterthought (first attempt at contributing to MRI).